### PR TITLE
Update research.md

### PR DESCRIPTION
**Pull request by bstabs-alt**

### CODE REVIEW

File: research.md [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+        The range of features in the application are implemented by users to increase their productivity. 
+        Anti-social can block websites and apps such as Facebook to reduce distractions. 
+        You can also set a schedule for when the blocking will occur and for how long. 
+        Lastly there is a feature called locked mode where you cannot undo a session until it is over, which is useful if you try to end the session to procrastinate.  
+        </div>
+    </td>
+    <td style="width:50%">
+        <img style="float: right;" src="freedom.png">
+    </td>
+</table>
```
Comment (Aaron-Rumpler) [inline research.md:19]: Do we really want a table here. Also, you can do tables in Markdown, no need for HTML.

File: research.md [hunk 2]
```diff
@@ -1,32 +1,50 @@
 # What already exists that is similar to our idea?
 <div align="justify">Our app falls under the category of focus applications, which are apps for the purpose of reducing the user’s level of procrastination. 
 This category has several different approaches used by these apps such as blocking websites, giving external motivation for working, and suggesting whether a user really wants to procrastinate. 
-The app we are creating will certainly overlap with some of these focus apps and their features, because there must be a basic level of interference whether it be prompts or flat out blocking of certain websites and applications.
 </div>
```
Comment (Aaron-Rumpler) [inline research.md:4]: Do we really need to justify the text. Most stuff's just left-aligned and looks fine.

**Review by Aaron-Rumpler — CHANGES_REQUESTED** [review]

**Review by NickSheetz — APPROVED** [review]

**Review by Khrys-Samsonova — DISMISSED** [review]

**Review by Khrys-Samsonova — APPROVED** [review]

File: research.md [hunk 3]
```diff
@@ -1,32 +1,47 @@
...
-<div align="justify">This application follows a very basic concept of asking whether you wish to enter a website that would distract you. 
-This more lenient application focuses on making the user think about whether they want a distraction or whether they should continue to work. 
-The downside to this app is that it requires a higher level of self-control unlike some other focusing apps.
-</div>
-
-![freedom](mindful.png)
+<h1 align="center">What already exists that is similar to our idea?</h1>
+
+> -  Our app falls under the category of focus applications, which are apps for the purpose of reducing the user’s level of procrastination. 
+> - This category has several different approaches used by these apps such as blocking websites, giving external motivation for working, and suggesting whether a user really wants to procrastinate.
```
Comment (Aaron-Rumpler) [inline research.md:4]: Not sure why the bullet points are quoted?

File: research.md [hunk 4]
```diff
@@ -1,32 +1,47 @@
...
-![freedom](forest.png)
-
-## Mindful Browsing
-<div align="justify">This application follows a very basic concept of asking whether you wish to enter a website that would distract you. 
-This more lenient application focuses on making the user think about whether they want a distraction or whether they should continue to work. 
-The downside to this app is that it requires a higher level of self-control unlike some other focusing apps.
-</div>
-
-![freedom](mindful.png)
+<h1 align="center">What already exists that is similar to our idea?</h1>
```
Comment (Aaron-Rumpler) [inline research.md:1]: My recommendation is to not worry much about the formatting, the static site generator will deal with that when it converts it to HTML.

Comment (Aaron-Rumpler) [reply research.md:1]: If we want to justify or center text, we'd be better off doing it in the site template.

Comment (Aaron-Rumpler) [reply research.md:1]: I'd suggest doing something like #9.

**Review by Aaron-Rumpler — APPROVED** [review]

**Review by Khrys-Samsonova — APPROVED** [review]

**Review by NickSheetz — APPROVED** [review]

**Pull request merged**